app/templates/rest-api/config/options/all.xml

```diff
@@ -92,11 +92,20 @@
     </range>
   </constraint>
 
+  <constraint name="docFormat">
+    <range type="xs:string" facet="true" collation="http://marklogic.com/collation/codepoint">
+      <facet-option>limit=5</facet-option>
+      <facet-option>item-order</facet-option>
+      <facet-option>descending</facet-option>
+      <path-index>docFormat</path-index>
+    </range>
+  </constraint>
```

Comment:
When you load the sample-data, this facet doesn't get populated. It would be nice if it would, but afraid that means touching all 3000 docs in that sample-data zip.. Hmm, didn't I write a nodejs script to fix lat/long in them?